app/app/build.gradle.kts

```diff
@@ -13,7 +13,7 @@ android {
     }
 
     packaging {
-        resources.excludes.add("META-INF/DEPENDENCIES")
+        //resources.excludes.add("META-INF/DEPENDENCIES")
```

Comment:
What's this?

Reply:
With this line, the app wouldn't build on my system.
I can stash this as a local change if you want this to be unchanged 🤷🏼‍♂️

Reply:
I don't remember why I added this, probably because it wouldn't build without it, but whatever. I will uncomment it again if I need to.